doc/cabal-commands.rst

```diff
@@ -377,7 +377,7 @@ folder. By default the latest version of the package is downloaded, you can
 ask for a spefic one by adding version numbers
 (``cabal get random-1.0.0.1``).
 
-.. option:: -s, --source-repository[=head|this|...]]
+.. option:: -s[[head|this|...]], --source-repository[=[head|this|...]]
```

Comment:
Does it have to have double braces in `-s[[head|this|...]]`? I'd think that one pair is enough.

Reply:
I grabbed what was rendered as-is from the console.